.travis/install_salt.sh

```diff
@@ -43,11 +46,17 @@ configure_salt() {
     printf \
         "%s: copying Salt minion configuration from %s\n" \
         "${0}" "${TEMPORARY_CONFIG_DIR}"
-    sudo rm -rf /etc/salt
-    sudo mkdir -p /etc/salt
-    sudo cp "${FORCE_FLAG}" -- "${TEMPORARY_CONFIG_DIR}/minion" /etc/salt/minion
+    ${SUDO} rm -rf /etc/salt
+    ${SUDO} mkdir -p /etc/salt
+    ${SUDO} cp "${FORCE_FLAG}" -- \
+        "${TEMPORARY_CONFIG_DIR}/minion" /etc/salt/minion
 }
 
+SUDO=""
```

Comment:
Instead of duplicating this code, maybe we need a common.bash that these files include.